scripts/ci/build_and_upload_wheels.py

```diff
@@ -62,7 +62,7 @@ def __str__(self) -> str:
 
 
 def build_and_upload(
-    bucket: Bucket, mode: BuildMode, gcs_dir: str, target: str, compatibility: str, upload_gcs: bool
+    bucket: Bucket | None, mode: BuildMode, gcs_dir: str, target: str, compatibility: str, upload_gcs: bool
```

Comment:
we still pass `--upload-gcs` on non-contributor ci and that would then trickle down to creating `bucket` which is then used
before it would always create `bucket` which fails authentication then for contributor ci

Reply:
I meant just in the context of that Python function, not generally.